aiida/backends/manager.py

```diff
@@ -179,8 +179,8 @@ def is_database_schema_ahead(self):
         """
 
     @abc.abstractmethod
-    def get_schema_version_code(self):
-        """Return the code schema version."""
+    def get_schema_version_head(self):
+        """Return the latest schema version."""
```

Comment:
The docstring could do with some extra clarification, also using the word "head" in it like the method name.

Reply:
done